protos/feast/core/FeatureService.proto

```diff
@@ -54,7 +54,6 @@ message FeatureServiceMeta {
 
 message LoggingConfig {
   float sample_rate = 1;
-  google.protobuf.Duration partition_interval = 2;
```

Comment:
Do we need to reserve #2 or does it it not matter since it's never been used by anyone?

Reply:
It's never been used.